text/0148-sampling-probability.md

```diff
@@ -0,0 +1,805 @@
+
+We can encode the adjusted count directly as a floating point or
+integer number in the range [0, +Inf).  This is a conceptually easy
+way to understand sampling because larger numbers mean greater
+representivity.
+
+Note that it is possible, given this description, to produce adjusted
+counts that are not integers.  Adjusted counts are an approximatation,
+and the expected value of an integer can be a fractional count.
+Floating-point adjusted counts can be avoided with the use of
+integer-reciprocal inclusion probabilities.
+
+#### Encoding inclusion probability
+
+We can encode the inclusion probability directly as a floating point
+number in the range [0, 1).  This is typical of the Statsd format,
+where each line includes an optional probability.  In this context,
+the probability is also commonly referred to as a "sampling rate".  In
+this case, smaller numbers mean greater representivity.
+
+#### Encoding negative base-2 logarithm of inclusion probability
+
+We can encode the negative base-2 logarithm of inclusion probability.
+This restricts inclusion probabilities to powers of two and allows the
+use of small non-negative integers to encode power-of-two adjusted
+counts.  In this case, larger numbers mean exponentially greater
+representivity.
+
+#### Multiply the adjusted count into the data
+
+When the data itself carries counts, such as for the Metrics Sum and
+Histogram points.
+
+This technique is less desirable because while it preserves the
+expected value of the count or sum, the data loses information about
+variance.  This may also lead to rounding errors, when adjusted counts
+are not integer valued.
+
+### Trace Sampling
+
+Sampling techniques are always about lowering the cost of data
+collection and analysis, but in trace collection and analysis
+specifically, approaches can be categorized by whether they reduce
+Tracer overhead.  Tracer overhead is reduced by not recording spans
+for unsampled traces and requires making the sampling decision for a
+trace before all of its attributes are known.
+
+Traces are expected to be complete, meaning that a tree or sub-tree of
+spans branching from a certain root are expected to be fully
+collected.  When sampling is applied to reduce Tracer overhead, there
+is generally an expectation that complete traces will still be
+produced.  Sampling techniques that lower Tracer overhead and produce
+complete traces are known as _Head trace sampling_ techniques.
+
+The decision to produce and collect a sample trace has to be made when
+the root span starts, to avoid incomplete traces.  Then, assuming
+complete traces can be collected, the adjusted count of the root span
+determines an adjusted count for every span in the trace.
+
+#### Counting child spans using root span adjusted counts
+
+The adjusted count of a root span determines the adjusted count of
+each of its children based on the following logic:
+
+- The root span is considered representative of _adjusted count_ many
```

Comment:
The phrasing tripped me a bit, I would use adjusted_count (fixed width font) to indicate that we're referring to a single variable, which space separated italic does not convey.